cg/meta/workflow/fastq.py

```diff
@@ -147,6 +147,15 @@ def parse_header(line: str) -> dict:
 
         return fastq_meta
 
+    @staticmethod
```

Comment:
I think both of these can be moved to `MutantFastqHandler`

Reply:
done